index.bs

```diff
@@ -670,6 +686,8 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
     </div>
     </li>
 
+    <li><dfn>Focus distance</dfn> is a numeric camera setting that controls the focus distance of the lens.  The setting usually represents distance in millimeters to the optimal focus distance.</li>
```

Comment:
> I figured platform/device specific stuff like that should be left alone since the device reports its possible values anyway.

@dsanders11 Valid point. Though having no conversion seems unfortunate in two respects:
 * Inability to show units in the UI.
 * Inability to hotswap one camera for another, and expect settings to persist.

I suppose apps could write javascript to transpose a setting's relative position in the legal range to the corresponding setting for a device with a different range. Though I'd rather we give this problem to the UAs.